examples/index.js

```diff
@@ -8,10 +8,16 @@ class TypistExample extends React.Component {
 
   state = {
     renderMsg: false,
+    headerDrection: 1,
+    msgDrection: 1,
   }
 
   onHeaderTyped = () => {
-    this.setState({ renderMsg: true });
+    setTimeout(() => this.setState({ renderMsg: true,
+                    headerDrection: -this.state.headerDrection }), 1000);
```

Comment:
just a nit, can we place each object property in a newline (not sure why eslint didn't catch it)